apps/wallet/src/ui/app/hooks/useGetCoinBalance.ts

```diff
@@ -17,6 +18,9 @@ export function useGetCoinBalance(
         {
             enabled: !!address && !!coinType,
             refetchInterval: 4000,
+            onError: (error) => {
```

Comment:
This can end up being annoying, and maybe it's better to have a warning when we know that balances might be wrong. Was this a design decision from @mystie711 ?

Reply:
This error is not necessarily about balance; it's more of a network failure

Reply:
I understand that; I think the same can be said with a different type of error display. I don't see the difference if it is a toast or something else.

Reply:
The warning can be there until we have a successful response.

Also what will happen if we use this hook for more that one coin type at the same time. For example in tokens view and it fails for all? Is it going to show x number of toasts?

Reply:
`rpc.getBalance` is a single call. It returns the balance of all the coins associated with a given address. So no multiple toasts.

Reply:
I don't see a reason where we will use `useGetCoinBalance` instead of `useGetCoinBalances` where multiple coins are involved?

Reply:
I can remove the toast if that is the biggest issue and move it into the component.

Reply:
this hook no longer uses toast to display the error